src/commands/commit/withHook/index.js

```diff
@@ -26,4 +27,13 @@ export const registerHookInterruptionHandler = () => {
   })
 }
 
+export const cancelIfRebasing = (mode: 'hook' | 'client') =>
+  mode !== 'hook'
+    ? Promise.resolve()
+    : execa('git', ['branch']).then(({ stdout: branchNames }) => {
+        if (/no branch/.test(branchNames)) {
+          process.exit(0)
+        }
+      })
```

Comment:
Can you explain why you're looking to branch names ? That's why because on the `rebase` process you're not on a named branch but on a commit revision ? There's another way to check if we're on a rebase process ?

Reply:
Hi exactly during a rebase, the current branch is `no branch, rebasing <branch name>`, I suppose using git status could do as well